webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -48,15 +48,21 @@ const HostDetails = ({ match }) => {
   }, [match.params.id, dispatch]);
 
   useEffect(() => {
-    document.body.classList.add('transprenet-border');
+    //  This is a workaround for adding gray background inspiring pf4 desgin
+    //  TODO: delete it when pf4 layout (Page copmponent) is implemented in foreman
+    document.body.classList.add('pf-gray-background');
+    return () => document.body.classList.remove('pf-gray-background');
   }, []);
 
   const handleTabClick = (event, tabIndex) => {
     setActiveTab(tabIndex);
   };
   return (
     <>
```

Comment:
Maybe i'm missing something, but why not set the background class directly on this element instead of dynamically on the body?